rocketpy/plots/rocket_plots.py

```diff
@@ -104,6 +105,181 @@ def thrust_to_weight(self):
 
         return None
 
+    def draw(self, vis_args=None):
+        """Draws the rocket in a matplotlib figure.
+
+        Parameters
+        ----------
+        vis_args : dict, optional
+            Determines the visual aspects when drawing the rocket. If None,
+            default values are used. Default values are:
+            {
+                "background": "#EEEEEE",
+                "tail": "black",
+                "nose": "black",
+                "body": "dimgrey",
+                "fins": "black",
+                "motor": "black",
+                "buttons": "black",
+                "line_width": 2.0,
+            }
+            A full list of color names can be found at:
+            https://matplotlib.org/stable/gallery/color/named_colors
+        """
+        if vis_args is None:
+            vis_args = {
+                "background": "#EEEEEE",
+                "tail": "black",
+                "nose": "black",
+                "body": "dimgrey",
+                "fins": "black",
+                "motor": "black",
+                "buttons": "black",
+                "line_width": 2.0,
+            }
+
+        # Create the figure and axis
+        _, ax = plt.subplots(figsize=(10, 5))
+        ax.set_aspect("equal")
+        ax.set_facecolor(vis_args["background"])
+        ax.grid(True, linestyle="--", linewidth=0.5)
+
+        csys = self.rocket._csys
+
+        # Draw rocket body
+        reverse = csys == 1
+        self.rocket.aerodynamic_surfaces.sort_by_position(reverse=reverse)
+        y_tube, x_tube = [], []
+        for surface, position in self.rocket.aerodynamic_surfaces:
+            if isinstance(surface, (NoseCone, Tail)):
+                # Append the x and y coordinates of the surface shape_vec to the respective lists
+                x_tube.extend((-csys) * surface.shape_vec[0] + position)
+                y_tube.extend(surface.shape_vec[1])
+            if isinstance(surface, Fins):
+                pass
+
+        # Negate each element in the y_tube list using a list comprehension
+        y_tube_negated = [-y for y in y_tube]
+        plt.plot(
+            x_tube, y_tube, color=vis_args["body"], linewidth=vis_args["line_width"]
+        )
+        plt.plot(
+            x_tube,
+            y_tube_negated,
+            color=vis_args["body"],
+            linewidth=vis_args["line_width"],
+        )
+
+        # Get nozzle position (kinda not working for EmptyMotor class)
+        x_nozzle = self.rocket.motor_position + self.rocket.motor.nozzle_position
+
+        # Find the last point of the rocket
+        idx = -1 if csys == 1 else 0
```

Comment:
This variable `idx` could have a better name. Perhaps something like `end_index`. The comment description is good, but does not supersedes intention revealing names.